src/backend/joanie/tests/signature/test_backend_signature_dummy.py

```diff
@@ -106,10 +106,8 @@ def test_backend_dummy_signature_get_signature_invitation_link(self):
         self.assertIn(expected_substring, response)
 
         contract.refresh_from_db()
-        self.assertIsNotNone(contract.student_signed_on)
+        self.assertIsNone(contract.student_signed_on)
         self.assertIsNotNone(contract.submitted_for_signature_on)
-        # Check that an email has been sent
-        self._check_signature_completed_email_sent("student_do@example.fr")
```

Comment:
I think we should update the docstring because we send an email anymore with the latest modifications.
```python
        """
        Once we call the method for the invitation link, we should receive a dummy
        link to sign the file. The `student_signed_on` should be equal to None because we are waiting 
        for the student signature.
        """
```